libdnf5/comps/environment/environment.cpp

```diff
@@ -159,41 +159,42 @@ std::string Environment::get_order() const {
 
 
 std::vector<std::string> load_groups_from_pool(
-    libdnf5::solv::CompsPool & pool, Id environment_id, bool required = true) {
-    Solvable * solvable = pool.id2solvable(environment_id);
-    Offset offset;
-    if (required) {
-        offset = solvable->dep_requires;
-    } else {
-        offset = solvable->dep_suggests;
-    }
-
-    std::vector<std::string> groups;
+    libdnf5::solv::CompsPool & pool, std::vector<EnvironmentId> environment_ids, bool required = true) {
```

Comment:
Is there any reason to pass the vector by value and not by reference `const std::vector<EnvironmentId> & environment_ids` ?

Reply:
Fixed, thanks.